.travis.yml

```diff
@@ -1,19 +1,22 @@
 language: elixir
 
 elixir:
+  - 1.7
   - 1.6
   - 1.5
-  - 1.4
 
 otp_release:
-  - 20.1
+  - 21.1
+  - 20.3
   - 19.3
   - 18.3
 
 matrix:
   exclude:
     - elixir: 1.6
       otp_release: 18.3
+    - elixir: 1.5
+      otp_release: 21.1
```

Comment:
I think you need to add an exclusion for 1.7 and OTP < 19 here too. See https://hexdocs.pm/elixir/compatibility-and-deprecations.html#compatibility-between-elixir-and-erlang-otp

Reply:
Are you sure? All the tests currently pass @jherdman

Reply:
![image](https://user-images.githubusercontent.com/73386/50548941-2454e980-0c12-11e9-8d39-90a9b2946c56.png)